File: Python/Practice/E-Labsheet/lab7/3.py
n = int(input("Enter a number: "))
if n <= 0 or n > 26 :
    print("Invalid input, program terminates.")
else :
    i = 1
    while i <= n :
        done = ""
        j = 0 
        while j < i :
            ch = ord("A") + j
            done += (chr(ch))
            j += 1
        print(done)
        i+= 1    
    i = n-1
    while i >= 0 :
        j = 0
        while j < i :
            print(chr(ord("A") + j) , end = "")
            j += 1
        print()
        i-= 1

File: C/Lab/Lab6/3.c
int numPieces;
    char board[BOARD_SIZE][BOARD_SIZE];

   
    for (int i = 0; i < BOARD_SIZE; i++) {
        for (int j = 0; j < BOARD_SIZE; j++) {
            board[i][j] = ' ';
        }
    }

    printf("Enter the number of pieces: ");
    if (scanf("%d", &numPieces) != 1) {
        printf("Invalid input for number of pieces.\n");
        return 1;
    }

 
    for (int i = 0; i < numPieces; i++) {
        char piece;
        int x, y;
        printf("Enter piece %d (format: A(x, y)): ", i + 1);
        if (scanf(" %c(%d, %d)", &piece, &x, &y) != 3) {
            printf("Invalid input format. Please use the format A(x, y).\n");
            i--; 
            continue;
        }
        setPieceOnTable(board, piece, x, y);
    }

    // Print the board
    printf("  ");
    for (int i = 0; i < BOARD_SIZE; i++) {
        printf("%d ", i);
    }
    printf("\n");

    for (int i = 0; i < BOARD_SIZE; i++) {
        printf("%d|", i);
        for (int j = 0; j < BOARD_SIZE; j++) {
            printf("%c|", board[i][j]);
        }
        printf("\n");
    }

    return 0;